docs/manual-doc/functions-and-operators/set-operations.md

```diff
@@ -0,0 +1,166 @@
+
+Meanwhile, operation `UNION ALL` returns the union set with duplicated elements.
+
+```
+<left> UNION [DISTINCT | ALL] <right>
+```
+where `<left>` and `<right>` must have the same number of columns and pair-wise data types.
+
+### Example
+
+The following statement
+```
+GO FROM 1 OVER e1 \
+UNION \
+GO FROM 2 OVER e1
+```
+return the distinct neighbors vertex id of `1` and `2` (along with edge `e1`).
+
+While
+```
+GO FROM 1 OVER e1 \
+UNION ALL\
+GO FROM 2 OVER e1
+```
+returns all the neighbors vertex id of `1` and `2`, with all duplications.
+
+`UNION` can also work with the `YIELD`-syntax-returned properties. For example, let's suppose the results of the following two queries 
+
+```
+nebula> GO FROM 1 OVER e1 YIELD e1._dst AS id, e1.prop1 AS left.1, $$.tag.prop2 AS left.2 -- query 1
+==========================
+| id  | left.1 | left.2  |
+==========================
+| 104 |    1   |    2    |    -- line 1
+--------------------------
+| 215 |    4   |    3    |    -- line 3
+--------------------------
+
+nebula> GO FROM 2,3 OVER e1 YIELD e1._dst AS id, e1.prop1 AS right.1, $$.tag.prop2 AS right.2  -- query 2
+==========================
+| id  | right.1 | right.2 |
+==========================
+| 104 |    1    |    2    |    -- line 1
+--------------------------
+| 104 |    2    |    2    |    -- line 2
+--------------------------
+```
+
+And the following statement 
+
+```
+GO FROM 1 OVER e1 YIELD e1._dst AS id, e1.prop1 AS left.1, $$.tag.prop2 AS left.2
+UNION /* DISTINCT */
+GO FROM 2,3 OVER e1 YIELD e1._dst AS id, e1.prop1 AS right.1, $$.tag.prop2 AS right.2
+```
+
+will return as follows:
+```
+==========================
+| id  | left.1 | left.2 |    -- UNION or UNION DISTINCT. The column names come from query 1
+==========================
+| 104 |    1   |    2   |    -- line 1
+--------------------------
+| 104 |    2   |    2   |    -- line 2
+--------------------------
+| 215 |    4   |    3   |    -- line 3
+--------------------------
+```
+
+Notice that line 1 and line 2 return the same id (104) with different column values. The `DISTINCT` check duplication by all the columns for every line. So line 1 and line 2 are different.
+
+You can expect for the `UNION ALL` result
+```
+GO FROM 1 OVER e1 YIELD e1._dst AS id, e1.prop1 AS left.1, $$.tag.prop2 AS left.2
+UNION ALL
+GO FROM 2,3 OVER e1 YIELD e1._dst AS id, e1.prop1 AS right.1, $$.tag.prop2 AS right.2
+
+==========================
+| id  | left.1 | left.2 |    -- UNION ALL
+==========================
+| 104 |    1   |    2   |    -- line 1
+--------------------------
+| 104 |    1   |    2   |    -- line 1
+--------------------------
+| 104 |    2   |    2   |    -- line 2
+--------------------------
+| 215 |    4   |    3   |    -- line 3
+--------------------------
+```
+
+## INTERSECT
+
+Operation `INTERSECT` returns the intersection of two sets A and B (denoted by $A \bigcap B$), if the elements belongs both to set A and set B.
+
```

Comment:
Operation => operator? 
I think "_if the elements belongs both to set A and set B_" can be deleted. btw. belong